core/logger/Logger.cpp

```diff
@@ -267,14 +286,32 @@ void Logger::LoadConfig(const std::string& filePath) {
         logConfigInfo = "Load log config from " + filePath;
     } while (0);
 
+    // parse env log level
+    level::level_enum* envLogLevel = new(level::level_enum);
+    std::string aliyun_logtail_log_level = "";
```

Comment:
The naming style of this var does not accord to others.

Reply:
I will fix it.